### ISSUE TITLE
Rolling background image like Windows Te

### ISSUE THREAD

**Issue by spottedmahn** [microsoft/terminal#3900]
# Description of the new feature/enhancement
What about a set of rolling background images instead of a single one that Windows Terminal currently supports?  Looking at the same background all the time gets boring.

I like [Windows Themes](https://www.microsoft.com/en-us/store/collections/windowsthemes?SilentAuth=1&wa=wsignin1.0) w/ the desktop backgrounds slideshow feature.  I download a theme and I get a set of rolling background images on my desktop.  What about that same concept w/ Windows Terminal? Thoughts? 💭

### Screenshot reference of Windows 10:
![image](https://user-images.githubusercontent.com/14079228/70549061-9b447300-1b41-11ea-8ec9-f700b95b8e8e.png)

# Proposed technical implementation details (optional)
add a `"backgroundImages" : [ "file1.jpg", "file2.jpg" ]` property to `profiles`.

**Comment by DHowett-MSFT** [comment]
This isn't something we're likely to implement in version 1.0, but we would be willing to accept community contributions here. When we eventually have an extension model, this might make a good extension. Thanks!

**Comment by fde-capu** [comment]
Adding request for slowly panning big images; and option to configure a folder of images.

**Comment by carlos-zamora** [comment]
Lumping these feature requests into this issue too, since we need a consolidated story for how to handle multiple background images.

From https://github.com/microsoft/terminal/issues/14111#issue-1393159162:
> Introduce a keyword like `random` or a `setting` for your settings.json
or a `"pickBackgroundImage": [ "foo", "bar", "baz"]`

From https://github.com/microsoft/terminal/issues/7279#issue-678688656:
> I would like the ability to specify an array of filesystem paths in the `backgroundImage` property, in my profile(s), and have Windows Terminal randomly select one from the array upon initialization.

I'm also upgrading this issue to be the main issue for handling multiple background images. Seems like the desired scenarios so far are...
- [ ] use a new background image each time this profile is instantiated
- [ ] switch to a new background image every few minutes

**Comment by trustieee** [comment]
Any update on this?

**Comment by zadjii-msft** [comment]
Nope. We need to come up with a reasonable spec for settings to cover the scenarios outlines above, and then someone needs to actually write the code. If someone wants to help contribute a plan here, we'd be more than willing to help review 😄
